### PR TITLE
Toasts: add public util for hiding

### PR DESCRIPTION
**Pull request by Zedwag**

### CODE REVIEW

File: js/ui/toast.js [hunk 1]
```diff
@@ -1,301 +1,3 @@
...
-            default:
-                this.callBase(args);
-        }
-    }
-});
-
-registerComponent(WIDGET_NAME, Toast);
+import Toast from './toast/ui.toast';
 
 export default Toast;
```
Comment (ksercs) [inline js/ui/toast.js:3]: hm, maybe it's better to export utils here?

File: js/ui/toast/hide_toasts.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import $ from '../../core/renderer';
```
Comment (ksercs) [inline js/ui/toast/hide_toasts.js:1]: shouldn't we name this file just as `hide`?

File: js/ui/toast/hide_toasts.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+function hideToasts(container) {
+    const containerElement = $(container).get(0);
+    const toasts = ($('.dx-toast')).toArray();
```
Comment (ksercs) [inline js/ui/toast/hide_toasts.js:5]: let's use `container` in selector here to simplify code below

File: testing/tests/DevExpress.ui/hideToasts.tests.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
+import $ from 'jquery';
+import notify from 'ui/notify';
+import fx from 'animation/fx';
+import hideToasts from 'ui/toast/hide_toasts';
+
+QUnit.testStart(function() {
+    const markup =
+        '<div id="container"></div>';
```
Comment (ksercs) [inline testing/tests/DevExpress.ui/hideToasts.tests.js]: excess linebreak to me

File: testing/tests/DevExpress.ui/hideToasts.tests.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+            container: '#container',
+            position: 'bottom left',
+            displayTime: 100,
+        });
+
+        notify({
+            position: 'bottom right',
+            displayTime: 100,
+        });
+
```
Comment (ksercs) [inline testing/tests/DevExpress.ui/hideToasts.tests.js:30]:
```suggestion
```

File: testing/tests/DevExpress.ui/hideToasts.tests.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+        '<div id="container"></div>';
+
+    $('#qunit-fixture').html(markup);
+});
+
+QUnit.module('hideToasts', {
+    beforeEach: function() {
+        fx.off = true;
+        this.clock = sinon.useFakeTimers();
+
```
Comment (ksercs) [inline testing/tests/DevExpress.ui/hideToasts.tests.js:19]:
```suggestion
const displayTime = 100;
```

to simplify configs below

File: testing/tests/DevExpress.ui/hideToasts.tests.js [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+    },
+
+    afterEach: function() {
+        this.clock.tick(100);
+        fx.off = false;
+        this.clock.restore();
+    }
+}, () => {
+    QUnit.test('without argument hides all Toasts', function(assert) {
+        assert.equal($('.dx-toast').length, 2);
```
Comment (ksercs) [inline testing/tests/DevExpress.ui/hideToasts.tests.js]: lets create `getToast` method in beforeEach to get rid of copypaste

File: testing/tests/DevExpress.ui/hideToasts.tests.js [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+        hideToasts();
+
+        assert.equal($('.dx-toast').length, 0);
+    });
+
+    QUnit.test('with argument defined as string hides Toasts with corresponding container', function(assert) {
```
Comment (ksercs) [inline testing/tests/DevExpress.ui/hideToasts.tests.js]:
```suggestion
    QUnit.test('with string parameter hides Toasts with corresponding container', function(assert) {
```

File: testing/tests/DevExpress.ui/hideToasts.tests.js [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+        assert.equal($('.dx-toast').length, 2);
+
+        hideToasts('#container');
```
Comment (ksercs) [inline testing/tests/DevExpress.ui/hideToasts.tests.js]: let's create `$container` prop in beforeEach to make the code more declarative

**Review by ksercs — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Pull request merged**